FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Logging/DiagnosticLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Logging
+{
+    public class DiagnosticLogger : IDiagnosticLogger
```

Comment:
Feel users don't know where errors happened from this Logger? (class, function, etc.)

Reply:
Customers should not know the specific class information.  Log message can contain more information about failed function.